components/ItemCard.stories.ts

```diff
@@ -19,5 +19,6 @@ export const ItemCardDefault: Story = {
     postcode: 'E12',
     postable: true,
     itemId: 2,
+    reserved: true,
```

Comment:
✨  Yes! Adding to the component library !!

Let's push our use of stories though and make it into it's own example.

If you look at the `ConversationCard` stories you'll see it has a few different stories to show all the different states it might have in the app. Can you revert the default ItemCard to not being reserved and then add a `ItemCardReserved` story?

The other addition that will help loads is to add an interaction test that verifies the card with the `reserved` value is in fact showing the "reserved" tag in the UI. You can see how this is done by looking at the `play` function in the `ConversationCard` stories.

For this case, you will only need the `play` function in the new story export, not the `meta` object.

Reply:
I never really played around with Storybook so more than happy to do this.
It sounds pretty straightforward, but I will get back to you in case I have issues with anything.

Reply:
There are a few examples in the code base already of how we're using it and their documentation is pretty good, but in case of any doubt I'm happy to lend a hand 👌